localization/i18n/es/OrcaSlicer_es.po

```diff
@@ -4906,19 +4913,19 @@ msgid "Flow rate test - Pass 2"
 msgstr "Test de Caudal - Paso 2"
 
 msgid "Flow rate"
-msgstr "Test de caudal"
+msgstr "Test de Caudal"
 
 msgid "Pressure advance"
 msgstr "Avance de Presión Lineal"
 
 msgid "Retraction test"
-msgstr "Test de retracción"
+msgstr "Test de Retracciones"
 
 msgid "Orca Tolerance Test"
 msgstr "Test de Tolerancia Orca"
 
 msgid "Max flowrate"
-msgstr "Máximo caudal"
+msgstr "Test de Caudal Máximo"
```

Comment:
Caudal Máximo, test word is not added in the original string.

Reply:
Maybe Softfever should add it. Its a test.